src/containers/OnBoarding/onBoardOpotions.tsx

```
@@ -0,0 +1,659 @@
+  CyDTouchView,
+  CyDView,
+} from '../../styles/tailwindComponents';
+import bs58 from 'bs58';
+import Loading from '../../components/v2/loading';
+import { AnalyticEvent, logAnalyticsToFirebase } from '../../core/analytics';
+import useWeb3Auth from '../../hooks/useWeb3Auth';
+
+const enum ProviderType {
+  ETHEREUM = 'ethereum',
+  SOLANA = 'solana',
+}
```

Biome (1.9.4)

[error] The enum declaration should not be const

Const enums are not supported by bundlers and are incompatible with the 'isolatedModules' mode. Their use can lead to import inexistent values.
See TypeScript Docs for more details.
Safe fix: Turn the const enum into a regular enum.


(lint/suspicious/noConstEnum)